services/models.py

```diff
@@ -65,6 +68,15 @@ class Meta:
             ("can_view_sensitivedata", "Can view sensitive data"),
         )
 
+    def save(self, *args, **kwargs):
+        # Convenience for saving Services with only service_type and no name.
+        # When service_type is removed from the code base, this should be
+        # removed as well and every Service creation requires a name at that point.
+        if not self.name and self.service_type:
+            self.name = self.service_type.name
```

Comment:
Here the name used is ServiceType.name, but in the migration it's ServiceType.value. Should they both be the same?

Reply:
Yes, definitely. Good catch 👍